redash/query_runner/azure_sql_data_warehouse.py

```diff
@@ -0,0 +1,192 @@
+import json
+import logging
+import sys
+import uuid
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+logger = logging.getLogger(__name__)
+
+try:
+    import pyodbc
+    enabled = True
+except ImportError:
+    enabled = False
+
+# from _mssql.pyx ## DB-API type definitions & http://www.freetds.org/tds.html#types ##
+types_map = {
+    1: TYPE_STRING,
+    2: TYPE_BOOLEAN,
+    # Type #3 supposed to be an integer, but in some cases decimals are returned
+    # with this type. To be on safe side, marking it as float.
+    3: TYPE_FLOAT,
+    4: TYPE_DATETIME,
+    5: TYPE_FLOAT,
+}
```

Comment:
Are you sure the types_map stays the same? It usually changes between DB-API drivers.

Reply:
It seems to be same.
I had tested both pymssql and pyodbc.
Integers, double, varchar, nvarchar, bigint, and UUID.
And it worked.

So I will use imports from  mssql.